Sources/RegexBuilder/Anchor.swift

```diff
@@ -226,3 +226,64 @@ public struct NegativeLookahead<Output>: _BuiltinRegexComponent {
     self.init(_RegexFactory().negativeLookaheadNonCapturing(component()))
   }
 }
+
+/// A regex component that allows a match to continue only if its contents
+/// match at the given location.
+///
+/// A lookbehind is a zero-length assertion that its included regex matches at
+/// a particular position. Lookbehinds do not advance the overall matching
+/// position in the input string — once a lookbehind succeeds, matching continues
+/// in the regex from the same position.
+@available(SwiftStdlib 5.7, *) // TODO: How should this be gated?
```

Comment:
We probably want to adjust the wording to denote the "behind" part of the assertion. CC @natecook1000